addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -4,6 +4,24 @@ import contentMenu from 'website.contentMenu';
 import weWidgets from 'wysiwyg.widgets';
 import {_t} from 'web.core';
 
+weWidgets.LinkPopoverWidget.include({
+    /**
+     * @override
+     */
+    start: function () {
+        // hide popover while typing on mega menu
+        if (this.$target.parents('.o_mega_menu').length) {
+            this.$target.on('keydown.link_popover', () => {
+                this.$target.popover('hide');
+                clearTimeout(this.inDebounce);
+                this.inDebounce = setTimeout(() => this.$target.popover('show'), 1500);
+            });
+        }
+
+        return this._super.apply(this, arguments);
```

Comment:
You could use the ES6 syntax here too: `this._super(...arguments)`

Reply:
Copy pasta spaghetti